Automattic/sensei
# Use from name/address on the email notifications

Resolves #[E-mail notifications are not using from and from_name#6953](https://github.com/Automattic/sensei/issues/6953)
 
## Proposed Changes
* Update email sender to use the from name and address on the email notifications
* Update the preview to show the from name and address on the email notifications

## Testing Instructions
1. Configure a SMTP server using a plugin
2. Set an email from name/address on the Sensei > Emails > Settings to use custom values.
3. Publish a Course
4. Enable the Welcome email
5. Assing a student to a course
6. Check if the generated email is using the correct from name/address
7.  Check if the email preview are showing the from name and address


## Pre-Merge Checklist
